### PR TITLE
Backfill capitalize names

### PR DESCRIPTION
**Pull request by brendanshean**
## WHAT
A previous [PR](https://github.com/empirical-org/Empirical-Core/pull/10941) will be restricting the `before_save :capitalize_name` callback so that it only runs when the name is changed.  This backfill will ensure that we run CapitalizeNames.capitalize on all users at least once.

## WHY
user.capitalize_names was recently updated to utilize a gem that handles capitalizing of names properly.  We'd like to ensure that this is run on all user names so that their names are properly capitalized.

## HOW
`rake users:back_fill_capitalize_names`

FYI, it took 24 minutes on staging.
bundle exec rake users:backfill_capitalize_names  629.65s user 24.22s system 45% cpu 23:55.67 total

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? | No.
Have you deployed to Staging? |  YES
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | N/A

### CODE REVIEW

File: services/QuillLMS/lib/tasks/temporary/users.rake [hunk 1]
```diff
@@ -90,4 +90,13 @@ namespace :users do
     User.where(id: user_ids).destroy_all
     puts "Deleted #{user_ids.count} users"
   end
+
+  task backfill_capitalize_names: :environment do
+    progress_bar = ProgressBar.new(User.count)
+
+    User.find_each do |user|
+      user.capitalize_name
```
Comment (anathomical) [inline services/QuillLMS/lib/tasks/temporary/users.rake:98]: Do we need a `save` call after the `capitalize_name` call?

**Review by anathomical — COMMENTED** [review]
Quick question about whether this actually works as intended.

File: services/QuillLMS/lib/tasks/temporary/users.rake (hunk 1 above)
Comment (brendanshean) [reply services/QuillLMS/lib/tasks/temporary/users.rake:98]: Great point.  This actually doesn't do anything at the moment.

**Comment by brendanshean** [comment]
After adding the `user.save` the rake task will now take in excess of a 14 days on staging.  Due to this cost, I'm inclined to not do the backfill.

**Pull request closed without merging**